cm_tools.drush.inc

```diff
@@ -0,0 +1,304 @@
+<?php
+
+/**
+ * Implementation of hook_drush_sql_sync_sanitize().
+ */
+function cm_tools_drush_sql_sync_sanitize($site) {
+  _cm_tools_drush_sql_sync_sanitize_user_module($site);
+
+  // Fetch list of all tables.
+  $all_tables = drush_sql_get_class()->listTables();
+
+  // All cache tables.
+  $queries = array();
+
+  foreach ($all_tables as $name) {
+    if ($name === 'cache' || strpos($name, 'cache_') === 0) {
+      $queries[] = "DELETE FROM $name";
+    }
+  }
+
+  if (!empty($queries)) {
```

Comment:
Tempted to spin them all out into separate functions.

Reply:
Could do, but might be a bit much? Not sure what we'd gain.
With your nice helper function (from comment below), this would be quite concise and tidy.

Reply:
Renamed the queries variables.